platform/linux/startup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if [ ! -d /app/.github ] || [ ! -d /home/user/.cache ]; then
```

Comment:
Can we move all docker related files and instructions to a `docker` directory? I think it is getting a bit cluttered now, and for local development of a graphics toolkit you would almost never use Docker, so while it is good to have it, it somewhat of a fringe use case.